packaging/test_irods_client_rest_cpp.py

```diff
@@ -350,6 +350,35 @@ def test_list_with_accoutrements(self):
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
 
+    def test_query_handles_case_insensitivity__issue124():
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                file_count = 100
+                case_insensitive = 0
+
+                dir_name = 'test_query_collection_directory'
+                lib.make_large_local_tmp_dir(dir_name, file_count, 1024)
```

Comment:
Why are multiple data objects needed for this test?

Reply:
Simply in imitation of test_query_with_limit_and_offset

Reply:
Got it. Let's use a single data object with a name containing upper and lower case letters. That makes the test clearer.